tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -60,22 +60,24 @@ func NewSession(credentialPath, credentialAccountID, region string) (*awssession
 	return session, nil
 }
 
-// CreateWindowsVM creates an Windows VM instance under the same VPC as the existing OpenShift cluster,
+// CreateWindowsVM creates a Windows instance under the same VPC as the existing OpenShift cluster with the following:
 // - attaches existing cloud-specific cluster worker security group and IAM to gain the same access as the linux
 // workers,
 // - uses public subnet,
-// - attaches public ip and a security group that allows traffic from VPC range and RDP access from user's IP,
+// - attaches public ip to allow external access,
+// - adds a security group that allows traffic from within the VPC range and RDP access from user's IP,
 // - uses given image id, instance type, and key name,
 // - creates a unique name tag for the instance using the same prefix as the OpenShift cluster name, and
 // - logs id and security group information of the created instance in 'windows-node-installer.json' file at the
 // resourceTrackerDir.
-// The CreateWindowsVM outputs RDP access information in the commandline interface.
-func (s *cloudProviderInfo) CreateWindowsVM(imageId, instanceType, keyName string) error {
+// The CreateWindowsVM takes in imageId, instanceType, and sshKey name to create the instance and outputs RDP access
```

Comment:
Like I mentioned in my previous review. Don't repeat what the function does here. Instead combine it with the original statement.